tensorboard/components/tf_tensorboard/BUILD

```diff
@@ -52,6 +52,7 @@ tf_web_library(
     visibility = ["//visibility:public"],
     deps = [
         "//tensorboard/plugins/audio/tf_audio_dashboard",
+        "//tensorboard/plugins/beholder/client_side:dashboard",
```

Comment:
I suspect I makes sense to rename client_side to match the main polymer component name like the other plugins, so that would be beholder_dashboard.  And actually, that component itself maybe should be renamed to tf-beholder-dashboard (with a `tf-` prefix) anyway.